### ISSUE TITLE
Tachiyomi 0.14.6 lockup and crash when attempting to install any extension

### ISSUE THREAD

**Issue by DrMacinyasha** [tachiyomiorg/tachiyomi#10034]
### Steps to reproduce

1. Perform fresh install of Tachiyomi using APK from website.
2. Open Tachiyomi. 
3. Tap on "Browse" on bottom navigation bar. 
4. Tap on the "Extensions" tab. 
5. Tap on the "Install" button next to literally any extension in the list. Language, 18+, etc., are irrelevant. 

### Expected behavior

Tachiyomi starts downloading the relevant extension's APK. Once download is complete, extension installation should proceed with a package manager window appearing for user confirmation of installing a new app on their device. 

### Actual behavior

Tachiyomi app became unresponsive, showing a highlight over the "Install" button that was tapped, and after several seconds, the app disappears and returns to the home screen. AKA, it crashes. No error, warning, alert, or other message appears. 

### Crash logs

--------- beginning of system
10-18 07:34:56.126 17954 17954 E ActivityThread: Failed to find provider info for downloads
--------- beginning of main
10-18 07:35:00.771 18249 18306 E chromium: [1018/073500.770795:ERROR:variations_seed_loader.cc(69)] Failed to open file for reading. Errno: 2
10-18 07:35:01.150 18249 18318 E SQLiteLog: (284) automatic index on C(manga_id)
10-18 07:35:01.150 18249 18318 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
10-18 07:35:30.281 18470 18530 E chromium: [1018/073530.281001:ERROR:variations_seed_loader.cc(69)] Failed to open file for reading. Errno: 2
10-18 07:35:30.709 18470 18500 E SQLiteLog: (284) automatic index on C(manga_id)
10-18 07:35:30.709 18470 18500 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
App version: 0.14.6 (standard, c615f4d, 101, 2023-04-16T15:14Z)
Android version: 14 (SDK 34)
Android build ID: UD1A.230803.041
Device brand: google
Device manufacturer: Google
Device name: husky
Device model: Pixel 8 Pro
Device product name: husky

### Tachiyomi version

0.14.6

### Android version

Android 14

### Device

Google Pixel 8 Pro

### Other details

Screen recording of crash available: https://drive.google.com/file/d/10RDLkcZRMyLWlGQPf_Tcv7krQAAK7v8v/view?usp=drivesdk

Tried switching between "Legacy" and "PackageManager" for the "Installer" option under the "Advanced"  settings, but this made no difference. Tried first importing a backup from my previous phone (Google Pixel Fold on Android 14) however that had no effect. 

Also looked through other bug reports regarding executive failing to download, but didn't find any that mentioned the app crashing, not giving an error, and/or not related to MIUI.

Device is a brand new, stock, unmodified, not-rooted Google Pixel 8 Pro. Currently connected to public Wi-Fi, with the Google One VPN active, but there is no change if I disable the VPN, switch to VPN'ing through my home router using UniFi Teleport, use the Google Fi VPN, or switch off Wi-Fi entirely and just run over 5G (Google Fi, AKA, T-Mobile US). Given this testing, and lack of issues manually downloading extensions from the Tachiyomi website, I'm left to conclude this is not a network issue triggering the crashing behavior. 

App has been given permission multiple times for full access to `/storage/emulated/0/Tachiyomi/downloads/` and `/storage/emulated/0/Tachiyomi/backup/` however this did not cause a change in whether the app crashed. App has also been granted all permissions possible under `(Android) Settings > Apps > Tachiyomi > Permissions`.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/docs/faq/general) and [troubleshooting guide](https://tachiyomi.org/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
I've had 3 other people try with a Pixel 8 Pro and none of them encountered the same behavior.

**Comment by DrMacinyasha** [comment]
### tl;dr, no longer able to reproduce, no known changes to phone status or configuration except using a different Wi-Fi network on the same ISP, however 5G (totally different network path) was impacted as well which suggests the network was not the cause.

Well, I'm not sue what's changed between today and yesterday, but I'm unable to reproduce the issue any longer. Tapping "Install" doesn't cause any errors or crashes, and extensions are installing normally. I guess this is just destined to be chalked up to "spontaneous auto-resolution."

I haven't made any changes to my phone beside a power off and on to install a screen protector. It's possible there was some background update to the Google back-end services, or some particular app in the Play Store auto-updated. I simply uninstalled Tachiyomi, reinstalled using the APK I had previously downloaded, and was able to install an extension both before and after I restored a backup from my old phone. The one other relevant change I can think of is the network environment: 

When I was having issues with the extensions, I was using a third-party's enterprise/corporate Wi-Fi network, which was pretty restrictive (blocking any attempts to access Private Internet Access's website or servers, the official Wireguard project page, or the official OpenVPN project pages, for example) using some Palo Alto Networks gateways. Even after downloading the installers over 5G on my phone, and copying them over to my PC via USB, the PAN firewalls would block the WG or OpenVPN traffic regardless of port or destination. The only exceptions to this were the Google One VPN (no idea about the protocol), and UniFi Teleport (Wireguard, but AFAIK uses a central Ubiquiti service to find the UniFi gateway the WG VPN is supposed to connect to, much like how Tailscale functions). On my PC I was able to connect to my home router and route traffic over it via an L2TP VPN.

Currently, I'm back on my home network which is all Ubiquiti UniFi gear (UDM-Pro for the gateway, 2nd-gen UniFi switches, and UniFi 6 Mesh APs). I have the UDMP blocking traffic to/from certain less-than-friendly countries as well as poor reputation IP addresses, and I'm also running Pi-hole to block a lot of ad and tracking data traffic, but otherwise have no restrictions in place.

(For a quick bonus: Both the location I was at, and my home ISP, are TDS Business Internet, although the remote location was in a different IP block since they had a static IP and I don't. So that would seem to negate any ISP-level shenanigans. The location also wasn't routing traffic to some centralized corporate office, as the static IP I was connecting from, and a traceroute, confirmed that the location's network egressed to the ISP locally. Meaning the on-site PAN firewalls were hooked up straight to the cable modem.) 

**However**, I'm hesitant to point any fingers at the Wi-Fi, networks, firewalls, or ISP for one simple reason: I still had the issue when I disconnected from Wi-Fi entirely, and was using Google Fi's 5G service directly (both with Google One VPN, UniFi Teleport, and without). The network path which Google Fi routes over is very different than what the local ISP was using, with none of the same network hops on different hosts I tried tracing to. Additionally, I was able to manually download the APKs for the extensions directly from the Tachiyomi website. Had the local network been interfering somehow, I would expect those to be blocked as well.

**Comment by arkon** [comment]
Were you potentially installing a bunch of apps at the time?

**Comment by DrMacinyasha** [comment]
@arkon Nope, in this case I didn't have any other apps being installed via the Play Store or otherwise.
